### PR TITLE
Arabidopsis annotations

### LINKED ISSUE #1320: AraTha annotations
**Short description of what you'd like added:**
The catalog would benefit from annotations for Arabidopsis.

**Rationale and considerations:**
In particular, we'd like to have this for the analysis2 pipeline and paper.

### PR DESCRIPTION
**Pull request by andrewkern**
here is a PR including the latest Arabidopsis annotations from Ensembl into the catalog. 

I have already pushed the relevant intervals files to AWS and set the SHA hashes in the `catalog/AraTha/annotations.py` file

this will close #1320

### CODE REVIEW

**Comment by andrewkern** [comment]
btw turned out there was a bit of URL drift in this PR that had to be fixed... Ensembl links for flies had changed-- something we will need to watch out for in the future

File: stdpopsim/catalog/AraTha/annotations.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import stdpopsim
+
+_species = stdpopsim.get_species("AraTha")
+
+_an = stdpopsim.Annotation(
+    species=_species,
+    id="araport_11_exons",
+    description="Ensembl Havana exon annotations on GRCh38",
```
Comment (petrelharp) [inline stdpopsim/catalog/AraTha/annotations.py:8]: I believe this is not GRCh38?

File: stdpopsim/catalog/AraTha/annotations.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    annotation_source="araport11",
+    annotation_type="exon",
+)
+_species.add_annotations(_an)
+
+# add CDS
+_an2 = stdpopsim.Annotation(
+    species=_species,
+    id="araport_11_CDS",
+    description="Ensembl Havana exon annotations on GRCh38",
```
Comment (petrelharp) [inline stdpopsim/catalog/AraTha/annotations.py:37]: ditto

File: tests/test_annotations.py [hunk 3]
```diff
@@ -203,6 +203,31 @@ def test_unknown_chromosome(self):
...
+    @classmethod
+    def setup_class(cls):
+        species = stdpopsim.get_species("AraTha")
+        cls.an = species.get_annotations("araport_11_exons")
+
+    def test_known_chromosome(self):
+        cm = self.an.get_chromosome_annotations("2")
+        assert isinstance(cm, np.ndarray)
+
+    def test_known_chromosome_prefix(self):
```
Comment (petrelharp) [inline tests/test_annotations.py:221]: this one does the same as the previous test - I think you're supposed to try it with `"chr2"`?

**Review by petrelharp — COMMENTED** [review]
Awesome! A few minor issues.

**Comment by andrewkern** [comment]
this is what i get for late night coding!! will fix these

**Pull request merged**